widgets/badgedlabel/org.eclipse.nebula.widgets.badgedlabel/src/org/eclipse/nebula/widgets/badgedlabel/BADGE_COLOR.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Akuiteo (http://www.akuiteo.com). All rights reserved. This program and the
+ * accompanying materials are made available under the terms of the Eclipse
+ * Public License v1.0 which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: Laurent CARON (laurent.caron at gmail dot com) - initial API and
+ * implementation
+ *******************************************************************************/
+package org.eclipse.nebula.widgets.badgedlabel;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.RGB;
+
+/**
+ * This enumeration provides a set of color for the BadgedLabel control
+ */
+public enum BADGE_COLOR {
+	BLUE(0, 123, 255, SWT.COLOR_WHITE), //

```

Comment:
This should probably use the default SWT color symbols or the RGB class instead of introducing a new API.

Reply:
Enumeration removed and introduce a new setPredefinedColor(int color) that uses predefined SWT colors (SWT.COLOR_BLUE, SWT.COLOR_GRAY...)

Reply:
If this breaks API then the major version number of the widget needs to be updated.

Reply:
It does not break API because this widget has never been commited.
Breaking API is  called in my company the "docker" effect

Reply:
I did not realize that. Thanks.